addons/auth_crypt/auth_crypt.py

```diff
@@ -22,7 +22,8 @@
 magic_md5 = '$1$'
 magic_sha256 = '$5$'
 
-openerp.addons.base.res.res_users.USER_PRIVATE_FIELDS.append('password_crypt')
+from openerp.addons.base.res import res_users
```

Comment:
Hi, strangely the users are reporting this error: `AttributeError: 'module' object has no attribute 'base'`.
It would be nice to determine the conditions to reproduce this (it works just fine here), and explain how your patch makes a difference. Seeing that it still references `addons.base` directly it's difficult to see how it would not trigger the same error. How do you reproduce it? Thanks!